samples/subsys/display/cfb_shell/boards/native_posix.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SDL_DISPLAY_DEFAULT_PIXEL_FORMAT_MONO01=y
```

Comment:
This is doubly problematic, we should always try to avoid board specific overlays because it adds maintenance overhead and it is not the correct way, it should be done by the app, something like https://github.com/zephyrproject-rtos/zephyr/blob/1c8f1cd590e10dcc349ebd8649c313293e29209c/samples/subsys/display/cfb/src/main.c#L26-L29 should be added to https://github.com/zephyrproject-rtos/zephyr/blob/11a2107d991b14a7abb7081498c914a9f57ed089/subsys/fb/cfb_shell.c#L410-L429

Reply:
It' make a sense. I fixed it.